src/App.tsx

```diff
@@ -41,24 +43,33 @@ function BridgeApp(props: Props) {
   }, []);
 
   // Check if Network has been started
-  if (!isConnected(netClient)) {
+  if (!isConnected(net.client)) {
     return <p style={{ textAlign: 'center' }}>Connecting Network</p>;
   }
 
+  // Ensure that network is ropsten
+  if (net.network !== 'ropsten') {
```

Comment:
can we put this ('ropsten') in a config file that can be changed in different environments (eg: in future we may want to deploy a version that supports mainnet)

Reply:
It uses an environment value now with a fallback to `ropsten`